tests/providers/amazon/aws/transfers/test_dynamodb_to_s3.py

```diff
@@ -18,13 +18,12 @@
 from __future__ import annotations
 
 import json
-import unittest
 from unittest.mock import MagicMock, patch
 
 from airflow.providers.amazon.aws.transfers.dynamodb_to_s3 import DynamoDBToS3Operator
 
 
-class DynamodbToS3Test(unittest.TestCase):
+class DynamodbToS3Test:
     def setUp(self):
```

Comment:
For some reason keeping `setUp` not affect test execution `¯\_(ツ)_/¯`

```suggestion
class DynamodbToS3Test:
    def setup_method(self):
```

Reply:
I made the change anyways, wouldn't want that to bite us in the back later on ahah